frontend/src/screens/learner.tsx

```diff
@@ -252,54 +272,34 @@ export const MobileStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies })
 }
 
 export const DesktopStudyCards: FC<{studies: ParticipantStudy[]}> = ({ studies }) => {
-
-    const [displayArrows, setDisplayArrows] = useState<boolean>(false)
-    const viewport = useRef<HTMLDivElement>(null);
-
-    const checkOverflow = () => {
-        if (viewport.current) {
-            const overflow = viewport.current.scrollWidth > viewport.current.clientWidth;
-            return overflow
-        }
-        return false
-    };
-
     return (
-        <Stack justify='center' style={{ position: 'relative' }} 
-            onMouseOver={() => {
-                if(checkOverflow()){
-                    setDisplayArrows(true)
-                }
-            }} onMouseLeave={() => {
-                setDisplayArrows(false)
-            }}>
-            <ScrollArea viewportRef={viewport} type='never'>
-                <Flex align='center' justify='flex-start' gap='lg' pt='1rem' pb='2rem'>
-
-                    {studies.map(study => (
-                        <StudyCard key={study.id} study={study}/>
-                    ))}
-                                
-                </Flex>
-            </ScrollArea>
-
-            <div style={{ position: 'absolute', left: -10, cursor: 'pointer', marginTop: '-1rem', display: displayArrows ? 'block' : 'none' }}
-                onClick={() => {
-                    if(viewport.current){
-                        viewport.current.scrollBy({ left: -200, behavior: 'smooth' })
-                    }
-                }}>
-                <IconChevronLeft color={colors.purple} size='3.5rem'></IconChevronLeft>
-            </div>
-            <div style={{ position: 'absolute', right: -10, cursor: 'pointer', marginTop: '-1rem', display: displayArrows ? 'block' : 'none' }}
-                onClick={() => {
-                    if(viewport.current){
-                        viewport.current.scrollBy({ left: 200, behavior: 'smooth' })
-                    }
-                }}>
-                <IconChevronRight color={colors.purple} size='3.5rem'></IconChevronRight>
-            </div>   
-        </Stack>
+        <Box>
+            <Swiper
```

Comment:
@Coder-Srinivas can you remind me if we need swiper here still?

Reply:
It is not needed. The merge conflict was not properly resolved. It is fixed now.